src/core/ddsc/include/dds/ddsc/dds_public_status.h

```diff
@@ -219,7 +219,7 @@ dds_inconsistent_topic_status_t;
  */
 DDS_EXPORT dds_return_t
 dds_get_inconsistent_topic_status (
-  dds_entity_t topic,
```

Comment:
This really only takes a topic, so I think "topic" is the better choice.

Reply:
MISRA want the same argument names for all `dds__get_status` functions.
https://github.com/eclipse-cyclonedds/cyclonedds/blob/93ff10fc308b5f0ace361c2ce9b70da46fc21056/src/core/ddsc/src/dds__get_status.h#L23-L35

Reply:
Ah, of course ...

Maybe we can solve it by using `ent_type_` for the parameter name, though. Like:
```C
dds_return_t dds_get_##status_##_status (dds_entity_t ent_type_, dds_##status_##_status_t *status)
```

I haven't tried it, but I think it should work, because that's the bit in the type name after the `dds_` prefix, so `reader`, `writer`, `topic`, &c.

This whole macro violates MISRA rules 20.10, but that's only advisory so it doesn't matter 🙂

Reply:
Yeah, thank you it works 😎.